pkg/util/cli/deprecated.go

```diff
@@ -0,0 +1,81 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package cli
+
+import (
+	"fmt"
+	"io"
+	"strings"
+)
+
+func deprecatedFlagWarning(deprecated, replaceWith string) string {
+	return fmt.Sprintf("WARNING: `%s` argument is deprecated and will be removed in a future version. Please use `%s` instead.\n", deprecated, replaceWith)
+}
+
+type ReplaceFlag struct {
+	Hint string
+	Args []string
+}
+
+type ReplaceFlagFunc func(arg string, flag string) ReplaceFlag
+
+func ReplaceFlagPosix(arg string, flag string) ReplaceFlag {
+	newFlag := "-" + flag
+	return ReplaceFlag{
+		Hint: newFlag,
+		Args: []string{
+			strings.Replace(arg, flag, newFlag, 1),
```

Comment:
Since `FixDeprecatedFlags` use `strings.HasPrefix`, would returning `"-"+arg` be a bit more explicit ?